src/main/resources/upgrade-files/post-upgrade/sql/UAF-4199Remove_Discontinued_Outbound_Messaging_MOD.sql

```diff
@@ -0,0 +1,20 @@
+--=======================================================================================================================================
```

Comment:
So looks like this prop is on the table to be removed anyway (I know you know this, just wanted a history on github too). Merging now, thanks for your help here.